src/email.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I like the idea of using json for this sort of stuff, but I wonder if for stuff that's getting "compiled in" like this vs added at runtime we should still just use the equivalent Go constructs to get typechecking and refactoring (ie if we want to change Code again). 

If you want to do these as an example of the "resource" json-serialized format for sorta defining your application, I think we can, but maybe we could keep them out of the core aft repo and make separate repos for them (chasehensel/aft-datatypes) and have some way to load them in at runtime. WDYT?

Reply:
Could also use yaml as a compromise? I meant this to be an example of a resource, but also in core as they are commonplace useful